common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -121,6 +132,11 @@ define('video/09_events_plugin.js', [], function() {
             return player ? player.currentTime : 0;
         },
 
+        getCurrentLanguage: function() {
+            var player = this.state.lang;
```

Comment:
Yea, I think it make sense to call this something like  'language'.  In the `getCurrentTime` method, `player` is actually a video player object.  Here, `player` is a language code.